service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/DiagnosticReportMapper.java

```diff
@@ -211,4 +227,35 @@ private String buildNarrativeStatementForDiagnosticReport(DiagnosticReport diagn
             narrativeStatementTemplateParameters.build()
         );
     }
+
+    private String buildListOfHumanReadableNames(DiagnosticReport diagnosticReport) {
```

Comment:
If this function operates only on `List<DiagnosticReport.DiagnosticReportPerformerComponent>` then this should be the argument of this function

Reply:
The same thing in `buildNarrativeStatementForDiagnosticReport` where it uses `diagnosticReport.getIssuedElement()`